cycode/cli/files_collector/sca/maven/restore_gradle_dependencies.py

```diff
@@ -26,3 +27,6 @@ def get_lock_file_name(self) -> str:
 
     def verify_restore_file_already_exist(self, restore_file_path: str) -> bool:
         return os.path.isfile(restore_file_path)
+
+    def prepare_tree_file_path_for_command(self, manifest_file_path: str) -> str:
```

Comment:
pls use proper join path function. i don't think it will work good on Windows, for example

Reply:
what about this?:
manifest_file_path.replace('/' + BUILD_GRADLE_FILE_NAME, '/' + BUILD_GRADLE_KTS_FILE_NAME)

Reply:
@naftalicy I mean this function https://docs.python.org/3/library/os.path.html#os.path.join

Reply:
Im using replace now, I need to replace part of the string, join is not enough

Reply:
where do you use this method (`prepare_tree_file_path_for_command`)? I can not find any call of it

Reply:
as i can see was fixed. resolved